documentation/modules/auxiliary/admin/kerberos/get_ticket.md

```diff
@@ -0,0 +1,237 @@
+- **Verify** the TGT is correctly retrieved and stored in the loot
+- Try with the NT hash (`NTHASH` option) and the encryption key (`AESKEY`
+  option) instead of the password
+- Do: `run rhosts=<remote host> domain=<domain> user=<username> password=<password> action=TGS spn=<SPN>`
+- **Verify** the module uses the TGT in the cache and does not request a new one
+- **Verify** the TGS is correctly retrieved and stored in the loot
+- Do: `run rhosts=<remote host> domain=<domain> user=<username> password=<password> action=TGS spn=<SPN> KrbUseCachedCredentials=false`
+- **Verify** the module does not use the TGT in the cache and requests a new one
+- **Verify** both the TGT and the TGS are correctly retrieved and stored in the loot
+- Try with the NT hash (`NTHASH` option) and the encryption key (`AESKEY`
+  option) instead of the password
+
+## Options
+
+### DOMAIN
+The Fully Qualified Domain Name (FQDN). Ex: mydomain.local
+
+### USER
+The domain username to authenticate with.
+
+### PASSWORD
+The user's password to use.
+
+### NTHASH
+The user's NT hash in hex string to authenticate with. Not that the DC must
+support RC4 encryption.
+
+### AESKEY
+The user's AES key to use for Kerberos authentication in hex string. Supported
+keys: 128 or 256 bits.
+
+### SPN
+The Service Principal Name, the format is `service_name/FQDN` . Ex:
+cifs/dc01.mydomain.local. This option is only used when requesting a TGS.
+
+### KrbUseCachedCredentials
+If set to `true`, it looks for a matching TGT in the database and, if found,
+use it for Kerberos authentication. Default is `true`.
+
+## Scenarios
+
+### Requesting a TGT
+
+- TGT with NT hash
+
+```
+msf6 auxiliary(admin/kerberos/get_ticket) > loot
+
+Loot
+====
+
+host  service  type  name  content  info  path
+----  -------  ----  ----  -------  ----  ----
+
+msf6 auxiliary(admin/kerberos/get_ticket) > run verbose=true rhosts=10.0.0.24 domain=mylab.local user=Administrator nthash=<redacted> action=TGT
+[*] Running module against 10.0.0.24
+
+[+] 10.0.0.24:88 - Received a valid TGT-Response
+[*] 10.0.0.24:88 - TGT MIT Credential Cache saved on /home/msfuser/.msf4/loot/20221104181416_default_10.0.0.24_mit.kerberos.cca_912121.bin
+[*] Auxiliary module execution completed
+msf6 auxiliary(admin/kerberos/get_ticket) > loot
+
+Loot
+====
+
+host             service  type                 name  content                   info                                                                          path
+----             -------  ----                 ----  -------                   ----                                                                          ----
+10.0.0.24           mit.kerberos.ccache        application/octet-stream  realm: MYLAB.LOCAL, serviceName: krbtgt/mylab.local, username: administrator  /home/msfuser/.msf4/loot/20221104181416_default_10.0.0.24_mit.kerberos.cca_912121.bin
+
+msf6 auxiliary(admin/kerberos/get_ticket) > hosts
```

Comment:
The table alignment here for loot looks off. `mit.kerberos.ccache` should be under type, then `application/octet-stream` should be under content not name.

Having said that, again we're probably going to have all of this moved out of loot before this work is merged into master so I'd be in favor of just removing it for now so we don't forget about it and end up leaving outdated information in the final merge.